### PR TITLE
Deploying App Insights as Global Resource

### PR DESCRIPTION
**Pull request by Humoiz**
Moved App insights from regional deployment to global deployment

### CODE REVIEW

File: deployment/arm/region.json [hunk 1]
```diff
@@ -440,58 +424,6 @@
                 ]
             }
         },
-        {
-            "type": "Microsoft.ApiManagement/service/diagnostics",
```
Comment (aluong) [inline deployment/arm/region.json]: Why are we deleting the APIM child resources here?

Comment (wbreza) [reply deployment/arm/region.json]: Same question - How do we ensure the APIM instances are still hoooked up to AI ?

Comment (Humoiz) [reply deployment/arm/region.json]: Figuring it out. I tried to move these bits to global.json and used copy() function to create these resourced but it wont work as the deployment is per region.

File: deployment/arm/global.json [hunk 2]
```diff
@@ -30,6 +39,67 @@
...
+            "properties": {
+                "Application_Type": "web",
+                "Flow_Type": "Bluefield",
+                "Request_Source": "rest"
+            }
+        },
+        {
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2019-05-01",
+            "name": "[concat('nestedTemplate', copyIndex())]",
```
Comment (Humoiz) [inline deployment/arm/global.json]: Note, the copy is on a template level as we need to create all the resource within this template multiple times.

File: api/src/LinkyLink/Startup.cs [hunk 3]
```diff
@@ -61,8 +61,8 @@ public void ConfigureServices(IServiceCollection services)
                     options.JsonSerializerOptions.IgnoreNullValues = true;
                 });
 
-            // The following line enables Application Insights telemetry collection.
-            services.AddApplicationInsightsTelemetry(Configuration.GetSection("ApplicationInsights")["InstrumentationKey"]);
+            // Uncomment the following line to enable Application Insights telemetry collection.
+            //services.AddApplicationInsightsTelemetry(Configuration.GetSection("ApplicationInsights")["InstrumentationKey"]);
```
Comment (michaeldeongreen) [inline api/src/LinkyLink/Startup.cs]: I thought it was agreed upon to keep this line enabled?
https://github.com/Azure-Samples/openhack-production/pull/43#discussion_r375339060

Comment (Humoiz) [reply api/src/LinkyLink/Startup.cs]: Enabling it. It must have been commented out part of merging it with master. Glad you noticed it.

**Pull request merged**